Prometheus.HttpExporter.AspNetCore/HttpRequestDuration/HttpRequestDurationOptions.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace Prometheus.HttpExporter.AspNetCore.HttpRequestDuration
+{
+    public class HttpRequestDurationOptions : HttpExporterOptionsBase
+    {
+        public Histogram Histogram { get; set; } = Metrics.CreateHistogram(DefaultName, DefaultHelp, Histogram.ExponentialBuckets(0.0001, 1.5, 36), "code", "method", "action", "controller");
+
+        private const string DefaultName = "aspnet_http_request_duration";
```

Comment:
Standard metric naming style puts the units last, so  `aspnet_http_request_duration_seconds` would be better.